gitclone.adoc

```diff
@@ -9,18 +9,15 @@
 ////
 == Getting started
 
-The fastest way to work through this guide is to clone its Git repository and use the provided projects
-inside. To do this, run the following commands:
+Run the following command to clone the Git repository and use the projects provided
```

Comment:
I'm with YK on this one. We should keep "The fastest way to work ..." as it keeps the tone more conversational.

Reply:
Just a brief update. Can we go with what you suggested but add the "The fastest way..." back?

Something like:
"The fastest way to work through this guide is to clone the Git repository and use the projects provided inside"

Reply:
@Kubik42: I'm OK with "The fastest way to work through this guide..." because apparently it was there from the beginning. :) However, the grammar checker prefers this version:
The fastest way to work through this guide is to clone the Git repository and use the projects that are provided inside.

Reply:
@sishida ok that sounds good, can you also add this to your pull request? I think we are good to go in terms of merging. Thank you Sarah! :)